doc/releases/changelog-dev.md

```diff
@@ -96,6 +96,29 @@
        0.11917543, 0.08942104, 0.21545687], dtype=float64)
   ```
 
+* If the conditional does not include a mid-circuit measurement, then `qml.cond`
+  will automatically evaluate conditionals using standard Python control flow.
+  [(#6016)](https://github.com/PennyLaneAI/pennylane/pull/6016)
+
+  This allows `qml.cond` to be used to represent a wider range of conditionals:
```

Comment:
I got really excited because I encountered a case where I wished `qml.cond` worked with a classical value, but actually I needed the `Conditional` operator to work with a classical value 😅 Great improvement though 💯

Reply:
Out of curiosity, what were you doing that needed the classical support in conditional?

Reply:
Unfortunately I don't remember :(